### DISCUSSION TITLE
Assistance Needed with Locating SCAP Files

### DISCUSSION THREAD

**Discussion by fopson** [ComplianceAsCode/content#11845] [Q&A] [answered]
#### Description of problem:
Hello Everyone,

Apologies if this is a basic question, but I'm new to using this project. I discovered it while searching for SCAP files to import into Foreman. The Google search result titled "Security automation content in SCAP, Bash, Ansible, and other formats" led me here, but I'm having difficulty locating the SCAP/XML formats for the different profiles.

Could someone kindly direct me to the appropriate folder?

Thank you very much!

#### SCAP Security Guide Version:
All

#### Operating System Version:
All

**Comment by Mab879** [comment, marked as answer]
The SCAP files you need are in the releases. See the [releases](https://github.com/ComplianceAsCode/content/releases) page. The zip file attached to the release has the compiled SCAP files.

**Comment by fopson** [comment]
Thanks for the info @Mab879. I was able to find the SCAP files in the zip release file.
